std_msgs/Header header
geometry_msgs/Point[] points
float64[] velocity_profile
float64[] yaw_profile
